plugins/outputs/redistimeseries/README.md

```diff
@@ -0,0 +1,23 @@
+# RedisTimeSeries Producer Output Plugin
+
+The RedisTimeSeries output plugin writes metrics to the RedisTimeSeries server.
+
+```toml
+[[outputs.redistimeseries]]
+  ## The address of the RedisTimeSeries server.
+  address = "127.0.0.1:6379"
+
+  ## Password for logging-in to Redis
+  # password = ""
+
```

Comment:
Sorry, but this is still not in sync with your `sampleConfig`. I see the following when looking at the code in `plugins/outputs/redistimeseries/redistimeseries.go`

```suggestion
[[outputs.redistimeseries]]
  ## The address of the RedisTimeSeries server.
  address = "127.0.0.1:6379"

  ## Redis ACL credentials
  # username = ""
  # password = ""
  # database = 0
  
  ## Optional TLS Config
  # tls_ca = "/etc/telegraf/ca.pem"
  # tls_cert = "/etc/telegraf/cert.pem"
  # tls_key = "/etc/telegraf/key.pem"
  # insecure_skip_verify = false
```

Reply:
@srebhan I updated the README accordingly (small changes from yours), and similarly added the missing elements to the etc/telegraf.conf file in the correct location. Those items are identical copy+pasta, with each field matching  the _sampleConfig_ const in redistimeseries.go.  

Hopefully this is better ;)

Unfortunately I cannot edit the name of the PR to ensure it matches the Semantic Commit Message format. Can you?